File: Java/02/tasks-manager-application/src/main/java/org/tasksmanager/Repositories/TaskTypeStatDao.java
package org.tasksmanager.Repositories;

public class TaskTypeStatDao {
    private String typeCode;
    private int completed;
    private int running;

    public String getTypeCode() {
        return typeCode;
    }

    public void setTypeCode(String typeCode) {
        this.typeCode = typeCode;
    }

    public int getCompleted() {
        return completed;
    }

    public void setCompleted(int completed) {
        this.completed = completed;
    }

    public int getRunning() {
        return running;
    }

    public void setRunning(int running) {
        this.running = running;
    }
}

File: Net/02/TasksManager.Application/Repositories/TaskTypeStatDao.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TasksManager.Repositories
{
    public class TaskTypeStatDao
    {
        public String TypeCode { get; set; }
        public int Completed { get; set; }
        public int Running { get; set; }
    }
}
